src/hooks/UrqlUseMutation.re

```diff
@@ -22,12 +22,17 @@ external useMutationJs:
   (~query: string) => (useMutationResponseJs('a), executeMutation) =
   "useMutation";
 
-let useMutationResponseToRecord = (result: useMutationResponseJs('a)) => {
-  let data = result->dataGet;
+let useMutationResponseToRecord =
+    (parse: Js.Json.t => 'response, result: useMutationResponseJs('a)) => {
+  let data =
```

Comment:
Also similar to `useQuery` let's opt for `Belt.Option.map` like @gugahoa suggested rather than this `switch`.